packages/clarity-visualize/src/data.ts

```diff
@@ -52,7 +52,8 @@ export function metric(event: Data.MetricEvent): void {
         // Append region information to metadata
         for (let name in regions) {
             let r = regions[name];
-            let className = r === Layout.Interaction.Visible ? "visible" : (r === Layout.Interaction.Clicked ? "clicked" : Data.Constant.Empty);
+            let className = r.interactionState === Layout.Interaction.Clicked ? "clicked" : Data.Constant.Empty;
+            className += r.visibilityState === Layout.RegionVisibility.ScrolledToEnd ? "visible" : Data.Constant.Empty;
```

Comment:
Or Visible state as well? 
Shouldn't the scrolled to end get a separate class?
Also should input interactions get a separate class?

Reply:
This is mainly for visualization in devtools where we can easily see if a region was scrolled or not - adding too many classes will just confuse us when interpreting. so keeping it minimal with just visible and clicked. For the rest we can always look at the data.